integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -74,22 +74,22 @@ def __init__(self, user, enterprise_configuration):
     def _log_info(self, msg, course_or_course_run_key=None):
         LOGGER.info(
             generate_formatted_log(
-                self.enterprise_configuration.channel_code(),
-                self.enterprise_configuration.enterprise_customer.uuid,
-                None,
-                course_or_course_run_key,
-                msg
+                channel_name=self.enterprise_configuration.channel_code(),
+                enterprise_customer_uuid=self.enterprise_configuration.enterprise_customer.uuid,
```

Comment:
the indentation looks weird int he github UI on my screen but it seems to be find in the code itself.